docs/publishing.md

```diff
@@ -0,0 +1,48 @@
+## Versioning
```

Comment:
Not sure where to place this doc other than here.
Could consider displaying it in storybook like BM

Reply:
this works! I think convention is to capitalize docs file names -> `docs/PUBLISHING.md`